server/services/processReservoirRisk.js

```diff
@@ -0,0 +1,53 @@
+function getReservoirWetRisk (riskQueryResult) {
```

Comment:
Could these be condensed into 1 function potentially? Not sure if that would make things more complex to understand though so happy to forgo.